doc/develop/west/without-west.rst

```diff
@@ -111,6 +111,13 @@ done by setting the environment variable ``WEST_DIR`` to point to
 :file:`zephyrproject/.west/west` before running CMake to set up a
 build directory.
 
+Alternatively, the build system can be configured to call a custom script
+instead of ``west`` to handle targets such as ``flash`` and ``debug``. This can
+be done by setting the CMake ``FLASH_COMMAND`` variable to point to a custom
```

Comment:
Maybe provide a pointer, clue or example about defining CMake variables in general. Does not have to be long or detailed.

Reply:
Cool, added as well, let's see if it renders correctly